CITATION.cff

```diff
@@ -0,0 +1,18 @@
+cff-version: 1.2.0
+
+type: conference-paper
```

Comment:
don't we want this off to point to this repo or a release of Morpheus instead of the paper?